source/modules/MEI.neumes.xml

```diff
@@ -185,6 +177,42 @@
       <memberOf key="att.sylText"/>
     </classes>
   </classSpec>
+  <classSpec ident="att.neumetype" module="MEI.neumes" type="atts">
```

Comment:
Following other nomenclature this should probably be `neumeType` ?

Reply:
```suggestion
  <classSpec ident="att.neumeType" module="MEI.neumes" type="atts">
```

Reply:
@kepper

Reply:
yes probably it would be best to leave this as is and fix in MEI 6